### PR TITLE
Secure Payment Confirmation shipping in Android Chrome blog post

### PR DESCRIPTION
**Pull request by agektmr**
Secure Payment Confirmation shipping in Android Chrome blog post.

### CODE REVIEW

**Comment by agektmr** [comment]
cc: @stephenmcgruer

File: site/en/blog/spc-on-android/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+{% Aside %}
+
+As SPC is specialised in payment authentication, you can use this API directly
+only if you are a credit card issuer, bank, or other payment service provider.
+Otherwise, please reach out to your credit card issuer, bank or payment service
+provider if you are interested in using this feature.
+
+{% endAside %}
+
+## Strong Authentication for payments
```
Comment (anusmitaray) [inline site/en/blog/spc-on-android/index.md:56]: Just one thing to double-check—are we using "Strong Authentication" as an official name—is that why we have it in title case?

I'm aware of [SCA](https://www.fca.org.uk/firms/strong-customer-authentication), is that the reference being made here? If not, should it not be "Strong authentication for payments"?

**Review by anusmitaray — APPROVED** [review]
Made minor edits and left a comment to just double-check something, but other than that LGTM

File: site/en/blog/spc-on-android/index.md (hunk 1 above)
Comment (agektmr) [reply site/en/blog/spc-on-android/index.md:56]: Chatted offline. We'll use "Strong **a**uthentication" to avoid conflicting with the SCA.

**Pull request merged**